vagrant/scripts/goDeps.sh

```diff
@@ -1,5 +1,11 @@
 export GOPATH=/home/vagrant/go;
 export PATH=$PATH:/usr/local/go/bin;
+
+# Server dependencies.
 go get -d github.com/dancannon/gorethink;
 go get -d github.com/codegangsta/negroni;
 go get -d github.com/gorilla/mux;
+
+# Go command line tools.
+echo "export PATH=$PATH:$GOPATH/bin;" >> /home/vagrant/.profile;
```

Comment:
The vagrant .profile works. I took it from what Nate did in bootstrap.sh. 

Line 2 is actually a different path. This one is for running go binaries, the line 2 one is for go itself.